numpy/compat/py3k.py

```diff
@@ -47,7 +47,12 @@ def asstr(s):
     return str(s)
 
 def isfileobj(f):
-    return isinstance(f, (io.FileIO, io.BufferedReader, io.BufferedWriter))
+    if isinstance(f, (io.FileIO, io.BufferedReader, io.BufferedWriter)):
```

Comment:
Let's remove one level of indentation via `not ...:`?

We should also have a mock-up test for the type of object you wish to support.

Reply:
ok